composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "blomstra/syndication",
+    "name": "ianm/syndication",
```

Comment:
I had to change this in order to get it installed on my dev instance. Should probably change this back?

Reply:
Yeah definitely needs to be reverted.

Reply:
I pushed the change back to blomstra